nsfc-plugin/src/main/java/org/osc/controller/nsfc/api/NeutronSfcSdnRedirectionApi.java

```diff
@@ -125,20 +338,40 @@ public InspectionHookElement getInspectionHook(String inspectionHookId) throws E
             return null;
         }
 
-        return null;  // TODO (Dmitry) Implement
+        FlowClassifier flowClassifier = this.osCalls.getFlowClassifier(inspectionHookId);
+
+        if (flowClassifier == null) {
+            LOG.warn("No inspection hook for id %s", inspectionHookId);
+            return null;
+        }
+
+        InspectionHookEntity retVal = new InspectionHookEntity(null, null);
```

Comment:
dont like initializing this with null, null. The constructor is intended to make sure the InspectionHookEntity is created with the 2 values always initialized. If this constructor is no longer useful, we should refactor